charts/mattermost-operator/values.yaml

```diff
@@ -0,0 +1,142 @@
+  livenessProbeInitialDelay: 200
+
+  ## Configuration for Mysql operator orchestrator
+  ##
+  orchestrator:
+    config:
+      ApplyMySQLPromotionAfterMasterFailover: false
+      Debug: false
+      BackendDB: sqlite
+      DetachLostReplicasAfterMasterFailover: true
+      DetectClusterAliasQuery: "SELECT CONCAT(SUBSTRING(@@hostname, 1, LENGTH(@@hostname) - 1 - LENGTH(SUBSTRING_INDEX(@@hostname,'-',-2))),'.',SUBSTRING_INDEX(@@report_host,'.',-1))"
+      DetectInstanceAliasQuery: "SELECT @@hostname"
+      DiscoverByShowSlaveHosts: false
+      FailMasterPromotionIfSQLThreadNotUpToDate: true
+      HTTPAdvertise: "http://{{ .Env.HOSTNAME }}-svc:80"
+      HostnameResolveMethod: none
+      InstancePollSeconds: 5
+      ListenAddress: ":3000"
+      MasterFailoverDetachReplicaMasterHost: true
+      MySQLHostnameResolveMethod: "@@report_host"
+      MySQLTopologyCredentialsConfigFile: "/etc/orchestrator/orc-topology.cnf"
+      ProcessesShellCommand: "sh"
+      RecoverMasterClusterFilters: ['.*']
+      RecoverIntermediateMasterClusterFilters: ['.*']
+      RecoveryPeriodBlockSeconds: 300
+      RemoveTextFromHostnameDisplay: ":3306"
+      RecoveryIgnoreHostnameFilters: []
+      RaftNodes: []
+      SQLite3DataFile: "/var/lib/orchestrator/orc.db"
+      SlaveLagQuery: "SELECT TIMESTAMPDIFF(SECOND,ts,NOW()) as drift FROM sys_operator.heartbeat ORDER BY drift ASC LIMIT 1"
+      UnseenInstanceForgetHours: 1
+      OnFailureDetectionProcesses:
+        - "/usr/local/bin/orc-helper event -w '{failureClusterAlias}' 'OrcFailureDetection' 'Failure: {failureType}, failed host: {failedHost}, lost replcas: {lostReplicas}' || true"
+        - "/usr/local/bin/orc-helper failover-in-progress '{failureClusterAlias}' '{failureDescription}' || true"
+      PostIntermediateMasterFailoverProcesses:
+        - "/usr/local/bin/orc-helper event '{failureClusterAlias}' 'OrcPostIntermediateMasterFailover' 'Failure type: {failureType}, failed hosts: {failedHost}, slaves: {countSlaves}' || true"
+      PostMasterFailoverProcesses:
+        - "/usr/local/bin/orc-helper event '{failureClusterAlias}' 'OrcPostMasterFailover' 'Failure type: {failureType}, new master: {successorHost}, slaves: {slaveHosts}' || true"
+      PostUnsuccessfulFailoverProcesses:
+        - "/usr/local/bin/orc-helper event -w '{failureClusterAlias}' 'OrcPostUnsuccessfulFailover' 'Failure: {failureType}, failed host: {failedHost} with {countSlaves} slaves' || true"
+      PreFailoverProcesses:
+      # as backup in case the first request fails
+        - "/usr/local/bin/orc-helper failover-in-progress '{failureClusterAlias}' '{failureDescription}' || true"
+
+    image:
+      repository: quay.io/presslabs/mysql-operator-orchestrator
+      tag: 0.4.0
+      pullPolicy: IfNotPresent
+
+  ## Configuration for Mysql operator persistent volume
+  ##
+  persistentVolume:
+    accessModes: [ReadWriteOnce]
+    size: 10Gi
+
+  topology:
+    user: "b3JjaGVzdHJhdG9y"
+    password: "Nnc2NHBhaGJzUA=="
```

Comment:
do we want these exposed?

Reply:
Yeap they are the default mySQLoperator values that the user can change.